packages/app/src/common/hooks/use-wallet.ts

```diff
@@ -176,29 +174,28 @@ export const useWallet = () => {
           name: appName as string,
         },
       });
-      const { defaultUsername } = currentIdentity;
+      const { username } = account;
       if (NODE_ENV === 'test') {
-        currentIdentity.defaultUsername = '';
+        account.username = '';
```

Comment:
:C mutation

Reply:
Sorry, I'll fix this. It's only for testing, the integration tests have to fetch and validate usernames. This is bad, you're right.